nextflow_schema.json

```diff
@@ -272,6 +272,29 @@
             "help_text": "Downstream analyses include a series of R scripts based on the Immcantation Alakazam, ChangeO and Shazam packages to calculate:\n- Clonal abundance and diversity\n- Clonal lineage tree export in graphML\n- Clonal overlap and statistics\n- Mutational load\n- Isotype and V-family distribution",
             "fa_icon": "fas fa-angle-double-right"
         },
+        "report_options": {
+            "title": "Report options",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "report_rmd": {
+                    "type": "string",
+                    "default": "/Users/gisela/Projects/Pipelines/nf-core-bcellmagic/assets/repertoire_comparison.Rmd",
```

Comment:
Is the "/Users/gisela/Projects/Pipelines/nf-core-bcellmagic" part of the path correct?  Similar below, in report_css and report_logo.

Reply:
Ah, thanks for the catch, I've updated it now